src/Command/AnalyseCommand.php

```diff
@@ -102,6 +103,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			$generateBaselineFile = 'phpstan-baseline.neon';
 		}
 
+		$allowEmptyBaseline = (bool) $input->getOption('allow-empty-baseline');
```

Comment:
This set to `true` should result in an error if `$generateBaselineFile` is null.

Reply:
Done. I'm not very satisfied with the error message, if you have a better idea...